### PR TITLE
Permission for Date parameter plugin

### PR DESCRIPTION
**Pull request by leejaycoke**
# Description

Hi there.
This is a PR for date parameter plugin permission.

Github repo: https://github.com/jenkinsci/date-parameter-plugin
Issue URL: https://issues.jenkins-ci.org/browse/HOSTING-349

# Permissions pull request checklist

### Always

- [O] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [O] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [O] Make sure the file is created in `permissions/` directory
- [O] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [O] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [O] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [O] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [O] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

Thanks.

### CODE REVIEW

File: permissions/plugin-date-parameter.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "date-parameter"
+paths:
+- "me/leejay/jenkins/date-parameter"
+developers:
+- "leejaycoke"
+---
```
Comment (daniel-beck) [inline permissions/plugin-date-parameter.yml]: This shouldn't be here.

Comment (leejaycoke) [inline permissions/plugin-date-parameter.yml]: I've found a problem. And it was fixed.
It's '---' string at last line.
But I do not know if "This" means this.

**Comment by leejaycoke** [comment]
@daniel-beck 
I've found a problem. And it was fixed.
It's '---' string at last line.
But I do not know if "This" means this.

**Comment by leejaycoke** [comment]
Thanks.

**Pull request merged**